lib/customer.rb

```diff
@@ -0,0 +1,52 @@
+# Pauline Chane (@PaulineChane on GitHub)
+# Ada Developers Academy C14
+# Grocery Store - customer.rb
+# 9/28/2020
+
+# imports
+require 'csv'
+
+# Class object spec file for Customer class.
+class Customer
+  # constants
+  ADDRESS_TEMPLATE = %i[street city state zip].freeze
+
+  # generic getters/setters
+  attr_accessor :email, :address
+  attr_reader :id
```

Comment:
You should make all of these reader methods instead of accessors.  You don't need to give users of your class the ability to directly modify the email or address